rpc/src/v1/impls/debug.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Debug APIs RPC implementation
+
+use std::sync::Arc;
+
+use ethcore::client::BlockChainClient;
+use transaction::LocalizedTransaction;
+
+use jsonrpc_core::Result;
+use v1::traits::Debug;
+use v1::types::{Block, Bytes, RichBlock, BlockTransactions, Transaction};
+
+/// Debug rpc implementation.
+pub struct DebugClient<C> {
```

Comment:
I think this mod needs to be enabled in `parity/rpc_apis.rs`? Or did I missed that?

Reply:
Oh, that's 100% true. Haven't tested it properly, will add to CLI and actually check if it's queryable through RPC.